src/cmd-run

```diff
@@ -114,7 +114,16 @@ if [ -z "${IGNITION_CONFIG_FILE:-}" ]; then
             }
         ]
     },
-    "storage": {},
+    "storage": {
+        "files": [
+            {
+                "filesystem": "root",
+                "path": "/etc/sysctl.d/10-coreos.conf",
+                "contents": { "source": "data:,kernel.printk%20=%203%204%201%207%0A" },
```

Comment:
really dumb question. It's ascii encoded and not base64 encoded. Is there a tool you use for that? for base64 it's easy with `base64` cli utility.

Reply:
I wrote it by hand like a caveman. :) I do prefer URL encoding for smaller strings like this since you can kinda make it out still. With base64 you basically have to pass it through to understand it. In this case, I'm open to just dynamically calling `base64` if you prefer.

Reply:
haha no no.. I was just wondering if there was an actual tool you used because I felt like I should know of one.